massa-protocol-worker/src/tests/ban_nodes_scenarios.rs

```diff
@@ -114,21 +125,11 @@ fn test_protocol_bans_node_sending_block_header_with_invalid_signature() {
                     .len(),
                 1
             );
-
-            (
-                network_controller,
-                protocol_controller,
-                protocol_manager,
-                consensus_event_receiver,
-                pool_event_receiver,
-                selector_event_receiver,
-            )
         },
     )
 }
 
 #[test]
-#[serial]
```

Comment:
removing "serial" does not cause any problem with other tests ?

Reply:
Tested and don't seems too CI is agree with me so we can stay like this :)